cristin-commons/src/main/java/no/unit/nva/cristin/model/Constants.java

```diff
@@ -16,7 +16,7 @@
     public static final String REL_NEXT = "rel=\"next\"";
     public static final String REL_PREV = "rel=\"prev\"";
 
-    public static final ObjectMapper OBJECT_MAPPER = JsonUtils.dtoObjectMapper;
+    public static final ObjectMapper OBJECT_MAPPER = JsonUtils.dynamoObjectMapper;
```

Comment:
Why are you doing this?

Reply:
Cristin does not  like empty json objects, this mapper skips empty arrays

Reply:
add comment for reason you are doing this, or better rename OBJECT_MAPPER to something else

Reply:
Switched back to dtoObjectMapper in Constants, using NON_EMPTY mapper only in CreateCristinProjectApiClient